pkg/kubectl/categories/categories.go

```diff
@@ -21,10 +21,15 @@ import (
 	"k8s.io/client-go/discovery"
 )
 
+// CategoryExpander interface provides a higher layer of group resource mapping.
+// category can be seen as a classification or 'tag' in the form of key string,
+// which can be used to retrieve a list of groupResource objects
+// 'schema.GroupResource' struct can be seen as a lower layer of group-resource mapping.
 type CategoryExpander interface {
 	Expand(category string) ([]schema.GroupResource, bool)
 }
 
+// SimpleCategoryExpander expands based on a hard-coded category list.
```

Comment:
SimpleCategoryExpander implements CategoryExpander interface using a static mapping of categories to GroupResource mapping.